resources/knative-serving/README.md

```diff
@@ -2,11 +2,12 @@
 
 ## Overview
 
-This chart includes [knative-serving](https://github.com/knative/docs/tree/master/docs/serving) release files.
+>**NOTE**: This is an experimental module.
+This chart includes [knative-serving CRDs](https://github.com/knative/docs/tree/master/docs/serving) release files.
```

Comment:
doesnt it include the real knative stuff?

Reply:
This will have the knative resources.